### PR TITLE
GAWB-1405

### PR DESCRIPTION
**Pull request by skwalker**
with https://github.com/broadinstitute/rawls/pull/649 and https://github.com/broadinstitute/firecloud-orchestration/pull/404

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/common/components.cljs
Comment (rushtong) [inline src/cljs/main/broadfcui/common/components.cljs]: Can refs

File: src/cljs/main/broadfcui/common/components.cljs
Comment (rushtong) [inline src/cljs/main/broadfcui/common/components.cljs]: Should this be a method in endpoints.cljs or are we getting away from centralizing urls there?

**Review by rushtong — APPROVED** [review]
👍 LGWG!

File: src/cljs/main/broadfcui/common/components.cljs
Comment (skwalker) [reply src/cljs/main/broadfcui/common/components.cljs]: for this one we need it to just be url string because it's trying to interact with bloodhound

File: src/cljs/main/broadfcui/common/components.cljs
Comment (rushtong) [inline src/cljs/main/broadfcui/common/components.cljs]: Suggest to sort and remove comment.

**Review by rushtong — COMMENTED** [review]
Needs a rebase. 
One minor suggestion, but fine as is.

File: src/cljs/main/broadfcui/common/components.cljs
Comment (davidangb) [inline src/cljs/main/broadfcui/common/components.cljs:686]: after discussion in planning - contrary to my comment in your other PR - we won't implement that yet. So, just delete this comment.

File: src/cljs/main/broadfcui/common/components.cljs
Comment (davidangb) [inline src/cljs/main/broadfcui/common/components.cljs:707]: I defer to PO, but I'd make this just "tag (count)", less verbose.

**Review by davidangb — COMMENTED** [review]
code looks good as TL. I'm waiting for rawls/orch to merge before testing this locally.

**Comment by davidangb** [comment]
Copying Slack discussion into the PR so we have a record of it:

if you copy a long-ish string of text - like "cantaloupe", or the name of the workspace bucket e.g. "fc-9bb00c6e-ba3b-4800-b17a-f9729042a8ff" ... then you attempt to paste that text into the tags input field, the text is cut off. It appears to be cut off at the expected width of a tag "badge".

This appears to be a bug in the bootstrap-tagsinput library. We can see the same bug at http://bootstrap-tagsinput.github.io/bootstrap-tagsinput/examples/.

A leading solution to this issue is removing bootstrap-tagsinput and using select2 instead; see https://select2.github.io/examples.html#tags. This is especially nice since we already have select2 in the app.

**Pull request closed without merging**
